src/services/messages-service.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
```

Comment:
Make sure to include documentation regarding what this service is testing.

Reply:
Not sure what you mean by testing. This is used to make the API calls for messages to work.
This is still prone to changes with the sockets work due.

Reply:
Regardless, added documentation in https://github.com/sabrasisler/FSE-Final-Project-React/pull/7 since my PRs are stacked.